crates/ty_python_semantic/resources/mdtest/generics/pep695/classes.md

```diff
@@ -133,6 +133,14 @@ reveal_type(Constrained[int | str]())  # revealed: Constrained[int | str]
 reveal_type(Constrained[object]())  # revealed: Unknown
 ```
 
+If the type variable has a default, it can be omitted:
+
+```py
+class WithDefault[T, U = int]: ...
+
+reveal_type(WithDefault[str]())  # revealed: WithDefault[str, int]
+```
```

Comment:
Is it worth also adding an assertion that explicitly shows that you can still specify a type argument for the typevar-with-default, and that we respect it if you do?

````suggestion
If the type variable has a default, it can be omitted:

```py
class WithDefault[T, U = int]: ...

reveal_type(WithDefault[str, str]())  # revealed: WithDefault[str, str]
reveal_type(WithDefault[str]())  # revealed: WithDefault[str, int]
```
````

Reply:
Good call!